src/fairchem/core/preprocessing/atoms_to_graphs.py

```diff
@@ -163,10 +164,16 @@ def convert(self, atoms: ase.Atoms, sid=None):
         """
 
         # set the atomic numbers, positions, and cell
+        positions = np.array(atoms.get_positions(), copy=True)
+        pbc = np.array(atoms.pbc, copy=True)
+        cell = np.array(atoms.get_cell(complete=True), copy=True)
+        positions = wrap_positions(positions, cell, pbc=pbc, eps=0)
```

Comment:
Why not just do `atoms.wrap()`?

Reply:
The wrapping uses an altered reduced cell instead of the original cell, and the energy/forces will be erased if the atoms object is modified. 

This version seems to work -- we could probably just use atoms.wrap as well.